dynawo/sources/Modeler/DataInterface/DYNCriteria.cpp

```diff
@@ -25,18 +25,48 @@ Criteria::Criteria(const boost::shared_ptr<criteria::CriteriaParams>& params) :
 
 Criteria::~Criteria() {}
 
+void
+Criteria::printAllFailingCriteriaIntoLog(std::vector<std::shared_ptr<FailingCriteria> > failingCriteriaVec,
+                                          const boost::shared_ptr<timeline::Timeline>& timeline,
+                                          double currentTime) const {
+  std::sort(failingCriteriaVec.begin(),
```

Comment:
I still don't understand why we sort it at the end instead of sorting it on the fly with a map